src/cli/lib/printer/printer.ts

```diff
@@ -72,4 +74,35 @@ export default class Printer {
 		const columns = process.stdout.columns || 80;
 		return Math.floor((columns - text.length) / 2);
 	}
+
+	total: number;
+	step: number;
+	load: loading.Loading;
+	constructor({total, step}: {total: number; step: number}) {
+		this.total = total;
+		this.step = step;
+		this.load = this.initLoader();
+	}
+
+	public initLoader(): loading.Loading {
+		return loading({
+			color: 'green',
+			interval: 100,
+			stream: process.stdout,
+			frames: ['◐', '◓', '◑', '◒'],
+		}).start();
+	}
+
+	public startStep(text: string, load: loading.Loading): any {
```

Comment:
Do we need to pass the `load` as a parameter if it's already a member of the class?